docs/administering/sample-config/nginx-example.conf

```diff
@@ -24,7 +24,7 @@ server {
 # Configuration for Sandstorm shell and apps, over HTTPS.
 server {
   listen 443;
-  server_name example.com *.example.com;
+  server_name example.com *.example.com ~^sandstorm-[^.]*\.example\.com$;
```

Comment:
Adding this regex isn't correct for our example. Arguably we should have another example showing how to wedge Sandstorm into a wildcard that also has non-Sandstorm things on it, by using the `sandstorm-*.example.com` trick. But we can't really merge that into this example without it being inconsistent.

If you revert the server_name changes I'm happy to merge the X-Real-IP changes!